examples/range-filters/index.js

```diff
@@ -0,0 +1,155 @@
+import { TextField } from '@material-ui/core';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import MUIDataTable from '../../src/';
+
+class Example extends React.Component {
+  render() {
+    const columns = [
+      {
+        name: 'Name',
+        options: {
+          filter: true,
+          filterList: [],
+          customFilterListRender: v => `Name: ${v}`,
+          filterOptions: { names: ['a', 'b', 'c', 'Business Analyst'] },
+        },
+      },
+      {
+        name: 'Title',
+        options: {
+          filter: true,
+          filterList: [],
+          customFilterListRender: v => `Title: ${v}`,
+          filterType: 'textField', // set filterType's at the column level
+        },
+      },
+      {
+        name: 'Location',
+        options: {
+          filter: false,
+        },
+      },
+      {
+        name: 'Age',
+        options: {
+          filter: true,
+          filterType: 'custom',
+          customFilterListRender: v => {
+            if (v['min'] && v['max']) {
+              return `Min Age: ${v['min']}, Max Age: ${v['max']}`;
+            } else if (v['min']) {
+              return `Min Age: ${v['min']}`;
+            } else if (v['max']) {
+              return `Max Age: ${v['max']}`;
+            }
+            return false;
+          },
+          filterOptions: {
+            names: [],
+            logic(age, filters) {
+              if (filters['min'] && filters['max']) {
+                return age < filters['min'] || age > filters['max'];
+              } else if (filters['min']) {
+                return age < filters['min'];
+              } else if (filters['max']) {
+                return age > filters['max'];
+              }
+              return false;
+            },
+            display: (filterList, onChange, index, column) => (
+              <div>
```

Comment:
To help keep the proliferation of new examples to the minimum needed, I'd like to see this added instead to the existing `customize-filters` example.

Also, if we could add

```
<div>
  <FormLabel>Age</FormLabel>
  <FormGroup row>
    // extra fields here
  </FormGroup>
</div>
```

the form group and label will help make it more clear what is happening in the example.

And lastly, we'll need to remove the `cellHeight` property from here https://github.com/gregnb/mui-datatables/blob/bbb1a27e27c1072e0897ada3fa4433a544f83e43/src/components/TableFilter.js#L311 to prevent the display from being truncated as will be possible now with some custom fields.

Reply:
Done